app/package.json

```diff
@@ -88,6 +88,7 @@
 		"front-matter": "4.0.2",
 		"html-entities": "2.3.2",
 		"jest": "27.4.7",
+		"json-to-graphql-query": "^2.2.4",
```

Comment:
This library doesn't seem to be maintained actively. Maybe we should roll our own implementation? (Feels like a fairly simple bit of logic to convert a JSON object to a known string format)

Reply:
Hmm just noticed that tags on GH are pretty outdated, and that there's only a single maintainer with unanswered issues/prs from months ago 🤷🏻 

Down to use it, but something to keep in mind 🙂